packages/discord-error/src/discord-error.ts

```diff
@@ -0,0 +1,133 @@
+export type ErrorBody = DiscordErrorBody | OAuthErrorBody;
```

Comment:
I don't actually know how licensing properly works but you should probably either append our license header or THIRD_PARTY_LICENSE.md it up

Reply:
For papertrail purposes, been talking about this on discord a lot

Reply:
Resolved!